themes/src/main/resources/theme/base/login/resources/js/webauthnRegister.js

```diff
@@ -0,0 +1,139 @@
+import { base64url } from "../../../common/keycloak/node_modules/rfc4648/lib/rfc4648.js";
+
+export function registerByWebAuthn(input) {
+
+    // Check if WebAuthn is supported by this browser
+    if (!window.PublicKeyCredential) {
+        document.getElementById("error").value = input.errmsg;
+        document.getElementById("webauth").submit();
```

Comment:
```suggestion
        returnFailure(input.errmsg);
        document.getElementById("error").value = input.errmsg;
        document.getElementById("webauth").submit();
```

Reply:
I fixed the codes as follows:

`        returnFailure(input.errmsg);`